test/UnitTests/MSTest.Core.Unit.Tests/Attributes/DynamicDataAttributeTests.cs

```diff
@@ -427,7 +410,6 @@ public string GetDynamicDataDisplayNameNonStatic(MethodInfo methodInfo, object[]
     /// <summary>
     /// The test method 1.
     /// </summary>
-    [FrameworkV2::Microsoft.VisualStudio.TestTools.UnitTesting.TestMethodAttribute]
```

Comment:
This is part of a `DummyClass`, I am not sure that we don't actually expect this to be a `TestMethod` for tests to work properly.